metricbeat/module/elasticsearch/_meta/Dockerfile

```diff
@@ -1,2 +1,2 @@
-FROM docker.elastic.co/elasticsearch/elasticsearch:6.4.3
-HEALTHCHECK --interval=1s --retries=300 CMD curl -f http://localhost:9200
+FROM docker.elastic.co/elasticsearch/elasticsearch:6.5.1
+HEALTHCHECK --interval=1s --retries=300 CMD curl -f http://localhost:9200/_xpack/license
```

Comment:
Would it also work with a more generic health check like `/_cluster/health`?

Reply:
Let me try this and see.

Reply:
I don't think it works reliably with `_cluster/health`. Jenkins CI passed but Travis CI failed with the same 404 error on `GET _xpack/license` that we were seeing earlier.

I think it's okay to base the health check on the URL we expect to call, i.e. `GET _xpack/license`, no?

Reply:
Yeah, it's fine, lets leave it with `_xpack/license` then.

Reply:
But lets take this into account https://github.com/elastic/elasticsearch/pull/35959 :(

Reply:
I'm going to fix all deprecated `_xpack` endpoints across the codebase in https://github.com/elastic/beats/pull/9656.